packages/usa-site-alert/src/styles/_usa-site-alert.scss

```diff
@@ -1,24 +1,25 @@
 @use "sass:math";
 @use "uswds-core" as *;
 
-// Alert variables ---------- //
-$site-alert-icons: (
-  info: "info",
-  emergency: "error",
-);
-
```

Comment:
> [!Note]
> Moved these icons to `alert-status-styles.scss` so that both alert and site alert have the same variants. This enables us to recommend users use the site alert markup structure for alerts that should share site margins.

Reply:
In the interest of performance, this change was reverted in [d3a2a08](https://github.com/uswds/uswds/pull/5636/commits/d3a2a083b4adb5545711acaccfcbaeeaaf278c82). Site alert and alert will keep the same variants they currently have in 3.8.2